pkg/metrics/metrics.go

```diff
@@ -188,6 +188,12 @@ func CounterForSubscription(name, installedCSV, channelName, packageName string)
 	return SubscriptionSyncCount.WithLabelValues(name, installedCSV, channelName, packageName)
 }
 
+func DeleteCSVMetric(oldCSV *olmv1alpha1.ClusterServiceVersion) {
+	// Delete the old CSV metrics
```

Comment:
Non-blocking nit: Please add comments to exported functions.

Reply:
Line 192 already has the comment? Did you have something else in mind?

Reply:
Updated the comment